docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -145,8 +145,9 @@ that is printed out.
 For the exact list of named groups ``@linter`` recognizes, see the `API
 documentation <https://api.coala.io/en/latest/>`__.
 
-For more info generally on regexes, see `Python re module
-<https://docs.python.org/3/library/re.html>`_.
+.. note::
+
+    Please refer to `Python3 re module <https://docs.python.org/3/library/re.html>`_ for understanding regular expressions and also refer `Dive into python <http://www.diveintopython3.net/regular-expressions.html>`_'s chapter on regexes.
```

Comment:
1. please break the line, it's too long ;)
2. --> refer **to** (for the second occurrence)
3. I think you can compact the sentence to:
```
Please refer to `Python3 re module <...>` and `Dive into python <...>` for information about regular expressions.
```